shared/teams/team/list.native.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import * as React from 'react'
+import {NativeSectionList} from '../../common-adapters/index.native'
+import type {Props} from './list'
+import {globalStyles} from '../../styles'
+
+type State = {
+  sections: Array<any>,
+}
+class List extends React.Component<Props, State> {
+  _renderSectionHeader = ({section}) => (section.key === 'body' ? this.props.renderRow(section.header) : null)
+  _renderRow = data => this.props.renderRow(data.item)
+  state: State = {sections: []}
+
+  static getDerivedStateFromProps(nextProps: Props) {
+    const rows = nextProps.rows || []
```

Comment:
instead of making the external props more complicated lets just translate it for native